server/plugin/api.go

```diff
@@ -285,6 +287,38 @@ func (p *Plugin) handleGetSubscriptions(w http.ResponseWriter, r *http.Request)
 	p.writeJSON(w, subscriptionList)
 }
 
+func (p *Plugin) handleSubscriptionNotifications(w http.ResponseWriter, r *http.Request) {
+	body, err := serializers.SubscriptionNotificationFromJSON(r.Body)
+	if err != nil {
+		p.API.LogError("Error in decoding the body for creating notifications", "Error", err.Error())
+		p.handleError(w, r, &serializers.Error{Code: http.StatusBadRequest, Message: err.Error()})
+		return
+	}
+
+	channelID := r.URL.Query().Get("channelID")
+	if channelID == "" {
+		p.API.LogError(constants.ChannelIDRequired)
+		p.handleError(w, r, &serializers.Error{Code: http.StatusBadRequest, Message: constants.ChannelIDRequired})
+		return
+	}
+
+	attachment := &model.SlackAttachment{
+		Text: body.DetailedMessage.Markdown,
+	}
+	post := &model.Post{
+		UserId:    p.botUserID,
+		ChannelId: channelID,
+	}
+
+	model.ParseSlackAttachment(post, []*model.SlackAttachment{attachment})
+	if _, err := p.API.CreatePost(post); err != nil {
+		p.API.LogError("Error in creating post", "Error", err.Error())
+	}
+
+	w.Header().Add("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
```

Comment:
If you do only this, the response body will be empty. You can do something like this - 
https://github.com/Brightscout/mattermost-plugin-servicenow/blob/9dc472a83c62dd840e462fb7743bdf88efd9577c/server/api.go#L497